Documentation/network/servicemesh/tls-cert.rst

```diff
@@ -36,7 +36,7 @@ Create TLS Certificate and Private Key
                 --namespace cert-manager \
                 --set installCRDs=true \
                 --create-namespace \
-                --set "extraArgs={--feature-gates=ExperimentalGatewayAPISupport=true}"
+                --set "extraArgs={--enable-gateway-api}"
```

Comment:
do we need to update the version at line 35 (i.e.  v1.10.0 to v1.15.0) ?

Reply:
Good point, I've updated the version. I've also changed `installCRDs` to `crds.enabled` since `installCRDs` has been deprecated in cert-manager 1.15.

Reply:
Thanks a lot for your kind help :100: :+1: :medal_military: